test/hocon_tests.erl

```diff
@@ -267,19 +269,70 @@ merge_when_resolve_test() ->
                         b => [#{k3 => 1}, #{k2 => 2}]}},
                  hocon:binary("a=[{k1=1}] [{k2=2}],a=[{k3=1}] [{k2=2}],b=${a}")).
 
-concat_error_test_() ->
-    [ ?_assertEqual({error, {concat_error, [{x, 1}, [1, 2]]}},
+concat_error_binary_test_() ->
+    [ ?_assertEqual({error,
+                     {concat_error,
+                      <<"failed to concat [{x,1},[1,2]] at_line 1">>}},
```

Comment:
nit: `faied_to_concat` ?

Reply:
also `failed to resolve` is fixed to `failed_to_resolve`